src/boot/bootctl-uki.c

```diff
@@ -143,21 +144,49 @@ static int read_pe_section(FILE *uki, const struct PeSectionHeader *section,
         return 0;
 }
 
+static int inspect_osrel(char *osrel, size_t osrel_size) {
+        _cleanup_fclose_ FILE *s = NULL;
+        _cleanup_free_ char *pname = NULL, *name = NULL;
+        int r;
+
+        assert(osrel);
+        s = fmemopen(osrel, osrel_size, "r");
+        if (!s)
+                return log_error_errno(errno, "Failed to open embedded os-release file, ignoring: %m");
```

Comment:
downgrade to warning please.

(we have this rule: if we ignore errors we hit, the log level should be LOG_WARNING at most. And the ", ignoring" suffix in the log message is usually a great indicator that it should be log_warning_errno() that is used)